docassemble/tclpgoogledocsmerger/data/questions/search_and_download.yml

```diff
@@ -78,6 +82,41 @@ code: |
     if help_link:
       selectable_fields[-1]['help'] = f'[More info here]({help_link})'
 ---
+code: |
+  # Take the interesting fields and make them 2 dimensional so easier to view in XLSX
+  stuff_to_snapshot = {
+     'start_time': str(start_time().format('yyyy-MM-dd')),
+     }    
+  # Get location
+  try:
+    import requests
+    response = requests.get(f"https://geolocation-db.com/json/{device(ip=True)}&position=true").json()
+    stuff_to_snapshot['country'] = response.get('country_code')
+    stuff_to_snapshot['state'] = response.get('state')
+    stuff_to_snapshot['city'] = response.get('city')
+    stuff_to_snapshot['latitude'] = response.get('latitude')
+    stuff_to_snapshot['longitude'] = response.get('longitude')
+  except:
+    stuff_to_snapshot['country'] = 'UNKNOWN'
+  # Don't let DA parse nameerrors so that all data is recorded in one block,
+  # regardless of how far in interview someone got
+  try:  
+    stuff_to_snapshot['all_clauses'] = comma_list(all_clauses)
```

Comment:
IMO we should grab the info returned by the CSV as well (instead of?) `all_clauses`. `all_clauses` is just the clauses that we were able to get from Drive, not all of the clauses that were found. I've already seen some issues where there are documents in the CSV that aren't on drive yet, or docs that were spelled wrong in the CSV. 

We should also start logging /fixing them separately, but for data completion it would help to have "what's really applicable to you", and "what people have been seeing", in case they drift.